examples/tornjak/keycloak/README.md

```diff
@@ -0,0 +1,74 @@
+# Deploy Tornjak with Auth Enabled
+
+To install Spire with the least privileges possible we deploy spire across 2 namespaces.
+
+- Install SPIRE CRDs and deploy SPIRE with Tornjak Enabled
+```shell
+# install SPIRE CRDs
+helm upgrade --install --create-namespace -n spire-mgmt spire-crds charts/spire-crds
+
+
```

Comment:
I suggest we first focus on installing the Keycloak and once this service is operational, and we know the values for `authServerURL`, either local or via ingress, we can put them into SPIRE with Tornjak configuration and deploy it